cuequivariance_torch/cuequivariance_torch/__init__.py

```diff
@@ -26,6 +26,7 @@
 from .primitives.transpose import TransposeSegments, TransposeIrrepsLayout
 
 from .primitives.equivariant_tensor_product import EquivariantTensorProduct
+from .primitives.segmented_polynomial_product import SegmentedPolynomialProduct
```

Comment:
Can we name it `SegmentedPolynomial` ? The notion of polynomial already contains the notion of product.

Reply:
done